Source/WebKit/UIProcess/Extensions/Cocoa/WebExtensionContextCocoa.mm

```diff
@@ -3143,7 +3143,7 @@ static _WKWebExtensionContextError toAPI(WebExtensionContext::Error error)
     auto *preferences = configuration.preferences;
     preferences._javaScriptCanAccessClipboard = hasPermission(_WKWebExtensionPermissionClipboardWrite);
 
-    if (purpose == WebViewPurpose::Background || purpose == WebViewPurpose::Inspector) {
+    if (purpose == WebViewPurpose::Inspector) {
```

Comment:
This change needs to apply to Inspector background pages too, they are the same situation. There can be multiple inspector background pages, so we will need multiple ActivityVariants.

I also think we need to keep setting `_hiddenPageDOMTimerThrottlingEnabled`, as these are hidden pages that we don't want throttled.